packages/tpetra/core/src/Tpetra_FECrsGraph_def.hpp

```diff
@@ -166,24 +243,21 @@ void FECrsGraph<LocalOrdinal, GlobalOrdinal, Node>::setup(const Teuchos::RCP<con
                                             std::runtime_error,"ownedPlusSharedRowMap must be locally fitted to the ownedRowMap");
      
    }
-
-   // For FECrsGraph, we do all the aliasing AFTER import.  All we need here is a constructor
-   if(ownedPlusSharedColMap.is_null())
-     inactiveCrsGraph_ = Teuchos::rcp(new crs_graph_type(ownedRowMap,0,StaticProfile,params));
-   else
-     inactiveCrsGraph_ = Teuchos::rcp(new crs_graph_type(ownedRowMap,ownedPlusSharedColMap,0,StaticProfile,params));
+ } else {
+   // We don't need the importer. Setting it to null helps detecting we are in a non-FE case later
```

Comment:
I'll improve the comment. Non-FE is indeed a poor word choice. I was trying to refer to the case where there's no need for an inactive graph, since owned and owned+shared row maps coincide.